src/widgets/notes/notes-summary.component.tsx

```diff
@@ -0,0 +1,193 @@
+import React, { useState, useEffect, Fragment } from "react";
```

Comment:
Can you post a picture?

Reply:
![NotesSummaryComponent](https://user-images.githubusercontent.com/28008754/75322859-b42d0c80-5884-11ea-8d1d-669d0cf012d3.gif)